src/reducers/statusReducer.js

```diff
@@ -71,6 +71,7 @@ export function requestedReducer(state = {}, { type, meta }) {
  * @param {object} action - Object containing the action that was dispatched
  * @param {string} action.type - Type of action that was dispatched
  * @param {string} action.path - Path of action that was dispatched
+ * @param action.meta
```

Comment:
Can we add the type as well? Seems like it would be object
```suggestion
 * @param {object} action.meta - Metadata for action
```

Reply:
fixed.